src/interpreter/index.ts

```diff
@@ -1308,9 +1314,8 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
                     this.execute(statement.body);
                 } catch (reason) {
                     if (reason instanceof Stmt.ExitForReason) {
-                        loopExitReason = reason;
                         break;
-                    } else {
+                    } else if (!(reason instanceof Stmt.ContinueForReason)) {
```

Comment:
These negated conditions in this file make the logic slightly confusing. It might be easier to understand if you had an empty if statement body with a comment explaining what should happen in this case? 
Perhaps something like this:
```typescript
if (reason instanceof Stmt.ExitForReason) {
    break;
} else if (reason instanceof Stmt.ContinueForReason) {
    //comment explaining what happens with `ContinueForReason`
} else {
    throw reason;
}

Reply:
Done